inc/File/PageResolver.php

```diff
@@ -26,7 +26,7 @@ public function resolveId($id, $rev = '', $isDateAt = false)
             $hash = '';
         }
 
-        if ($id !== '') {
+        if ($id !== '' && $id !== null) {
```

Comment:
Hmm, I wonder under which circumstances $id will be null. Maybe it would be worth to cast it to string

Reply:
As with most fixes in this PR, this is related to plugins calling this method incorrectly. I thought making sure the ID wasn't null here was a solution that would solve that problem for all the cases, but if that is not the case, I'm happy to make changes. I just don't understand what you mean by "Maybe it would be worth to cast it to string". Where would you like me to do that? At the beginning of the method? Sorry I just don't understand...

Reply:
Yeah a simple `$id = (string) $id;` at the beginning of the method should be enough